kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQueryHelper.java

```diff
@@ -67,6 +68,44 @@ public BigQuery connect(String projectName, String keyFilename) {
     }
   }
 
+  /**
+   * Returns a default {@link BigQuery} instance for the specified project with credentials provided
+   * in the specified file, which can then be used for creating, updating, and inserting into tables
+   * from specific datasets.
+   *
+   * @param projectName The name of the BigQuery project to work with
+   * @param keyFile The name of a file containing a JSON key that can be used to provide
+   *                    credentials to BigQuery, or null if no authentication should be performed.
+   * @param isJsonString If isJsonString is set to true, the method will interpret the keyFile as
+   *                     JsonString instead of file path
+   * @return The resulting BigQuery object.
+   */
+  public BigQuery connect(String projectName, String keyFile, Boolean isJsonString) {
```

Comment:
probably should use `boolean` so that the value can be `true` or `false` but not `null`.

Reply:
The BigQuerySinkConfig class will transform boolean properties to Boolean